src/fxcorrelator_xengops.py

```diff
@@ -358,14 +366,10 @@ def xeng_set_acc_time(corr, acc_time_s):
     if use_xeng_sim:
         raise RuntimeError('That\'s not an option anymore.')
     else:
-        new_acc_len = (corr.sample_rate_hz * acc_time_s) / (corr.xeng_accumulation_len * corr.n_chans * 2.0)
+        new_acc_len = (corr.sample_rate_hz * acc_time_s) / \
```

Comment:
Style pernicitiness: Can avoid explicit line continuation by wrapping the RHS in  (). Python style recomments not using \, since a misplaced space after the \ can change the meaning without it being apparent.